### DISCUSSION TITLE
Share your setups or your ideas

### DISCUSSION THREAD

**Discussion by EliTheSurfer** [EliTheSurfer/EliVim-VSCode#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share your setup.
  * Welcome others and are open-minded 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nadunindunil** [comment]
Hey @EliTheSurfer,

Thanks for sharing the setup.
A suggestion after going through the repo.

Can we keep only the relevant settings and plugins for the theme in files. As an example, `extensions.txt` got details not related to the theme, as well in the `settings.json` file. This will help users to focus on only the necessary settings.

Keep up the good work 👏

**Reply by EliTheSurfer** [reply]
Hello @nadunindunil , 
Thanks a lot for the feedback, really appreciate it!
You bring up a valid point. I agree that having only the relevant settings and plugins related to the *theme* and making vscode *keyboard only* would make things more straightforward for users. I’ll review the files and clean up the unnecessary details in extensions.txt and settings.json.
Thanks again for the suggestion and the kind words! I’ll get to work on this and keep you posted.

Cheers! 👏🌊
